src/Compiler/FSComp.txt

```diff
@@ -1733,4 +1733,5 @@ featureUnmanagedConstraintCsharpInterop,"Interop between C#'s and F#'s unmanaged
 3855,tcNoStaticMemberFoundForOverride,"No static abstract member was found that corresponds to this override"
 3859,tcNoStaticPropertyFoundForOverride,"No static abstract property was found that corresponds to this override"
 3860,chkStaticMembersOnObjectExpressions,"Object expressions cannot implement interfaces with static abstract members or declare static members."
-3861,chkTailCallAttrOnNonRec,"The TailCall attribute should only be applied to recursive functions."
\ No newline at end of file
+3861,chkTailCallAttrOnNonRec,"The TailCall attribute should only be applied to recursive functions."
+3862,parsStaticMemberImcompleteSyntax,"Incomplete static member declaration. Use the keywords 'static member' to declare a new member."
```

Comment:
It can also be `static let`, `static do`, or `static val`. I'd say it's probably good enough as is, but if you can come up with a simple text that wouldn't be strictly about `static member`, it could be good. 🙂